src/segments/azd.go

```diff
@@ -0,0 +1,77 @@
+package segments
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"strings"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type Azd struct {
+	props properties.Properties
+	env   platform.Environment
+
+	AzdConfig
+}
+
+type AzdConfig struct {
+	Version            int    `json:"version"`
+	DefaultEnvironment string `json:"defaultEnvironment"`
+}
+
+func (t *Azd) Template() string {
+	return " {{ .DefaultEnvironment }} "
+}
+
+func (t *Azd) Init(props properties.Properties, env platform.Environment) {
+	t.props = props
+	t.env = env
+}
+
+func (t *Azd) Enabled() bool {
+	var location string
+
+	folders := []string{".azure"}
```

Comment:
if for now this is one folder only, we can design for that today and still proceed to an array later. Unless if we allow the user to override this logic:

```suggestion
	folders := t.props.GetStringArray(LanguageFolders, []string{".azure"}) 
```